hasher-matcher-actioner/hmalib/lambdas/api/api_auth.py

```diff
@@ -1,20 +1,16 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
-import logging
+
 import os
+from hmalib.common import get_logger
 
 ACCESS_TOKEN = os.environ["ACCESS_TOKEN"]
```

Comment:
Is the plan to eventually store this as a secret in aws?

Reply:
Likely yes, I almost wrote that as a todo here. I am waiting to see what the frontend needs to have available before moving it though.